core/iwasm/libraries/wasi-nn/src/wasi_nn_tensorflowlite.cpp

```diff
@@ -216,7 +216,15 @@ tensorflowlite_init_execution_context(void *tflite_ctx, graph g,
                 NN_ERR_PRINTF("Error when enabling GPU delegate.");
                 use_default = true;
             }
-#elif defined(WASI_NN_ENABLE_EXTERNAL_DELEGATE)
+#else
+            NN_WARN_PRINTF("GPU not enabled.");
+            use_default = true;
+#endif
+            break;
+        }
+        case tpu:
+        {
+#if defined(WASI_NN_ENABLE_EXTERNAL_DELEGATE)
```

Comment:
We usually define macros in core/config.h, set them to 0, and use `cmake -DWAMR_BUILD_XXX=1` to set the macros to 1 (in build-scripts/config_common.cmake or other files), and then use `#if WASM_ENABLE_XXX != 0` in source code instead of `#ifdef` or `#if defined`. Not sure whether you want to modify `WASI_NN_ENABLE_GPU`, `WASI_NN_ENABLE_EXTERNAL_DELEGATE` in that way?

Reply:
done https://github.com/bytecodealliance/wasm-micro-runtime/pull/2447/commits/d7a3595f4794bdd81d28c5fa7400feb0347d9df5